sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -123,6 +130,7 @@ def update(
                     if collection_available:
                         logger.info(f"Collection {table_to_keep.name} already exists.")
                     else:
+                        # TODO: Enable dynamic schema option
```

Comment:
what does this mean?

Reply:
Milvus allows arbitrary additional fields including JSON with a `enable_dynamic_field=True` property in the CollectionSchema.
It would make adding new fields to a collection very easy, but I don't know if that really works the way Feast does. I believe if the FeatureView schema would change then the old collection would be flagged for deletion and a new one would be created.

Reply:
thanks for the explanation. Probably better to remove the to-do for now and have a spike in the future if that functionality is needed

Reply:
Sure, I had that note in to think about how we could potentially incorporate it but a spike ticket seems fine.